Assignment1/obsAvoidance/obsAvoidance.ino

```diff
@@ -0,0 +1,36 @@
+#include <Smartcar.h>
+
+const int forwardSpeed   = 70; 
+const int stoppingSpeed = 0; 
+const int stopDistance = 60;
+int distanceFromObject;
```

Comment:
Using global variables that are not constant (unless you have to) is not considered a good practice :fearful: